snapshots/commands/set_context.test.snap

```diff
@@ -26,5 +26,15 @@ exports[`set_context should set the context list branches selecting project from
   parent_id: br-parent-branch-123456
   created_at: 2021-01-01T00:00:00.000Z
   updated_at: 2021-01-01T00:00:00.000Z
+- id: br-numbered-branch-123456
```

Comment:
The change in branches tests caused snapshots in a completely different test to update.
That's not how it should be done.